controller/telemetry/server.go

```diff
@@ -274,13 +256,6 @@ func (s *server) ListPods(ctx context.Context, req *read.ListPodsRequest) (*publ
 			ControllerNamespace: controllerNS,
 			ControlPlane:        controllerComponent != "",
 		}
-		if added {
-			since := time.Since(updated)
-			item.SinceLastReport = &duration.Duration{
```

Comment:
if we're no longer populating `SinceLastReport`, we should deprecate / remove it from the api.